compose/cli/main.py

```diff
@@ -633,11 +633,13 @@ def pull(self, options):
         Options:
             --ignore-pull-failures  Pull what it can and ignores images with pull failures.
             --parallel              Pull multiple images in parallel.
+            --quiet                 Pull without printing progress information
```

Comment:
Thinking about this again, it does suppress *all* output, and it is essentially a pass-through for a flag named "silent", so --quiet seems appropriate.